Docs: markdown editor known issues (#8522)
ckeditor/ckeditor5: packages/ckeditor5-markdown-gfm/docs/features/markdown.md

The {@link module:markdown-gfm/markdown~Markdown} plugin allows switching the default CKEditor 5 output from HTML to Markdown. This allows for producing lightweight text documents with a simple formatting syntax, widespread among the programming and development communities and popular in many environments (e.g. GitHub). Coupled with the {@link features/autoformat autoformatting} feature, it allows for the full-fledged Markdown WYSIWYG editing experience.

> Please remember that Markdown syntax is very simple and therefore this plugin does not support rich-text output, even if the formatting maybe applied to the content.

Reviewer: ```suggestion
Please remember that Markdown syntax is very simple and it does not cover all the rich-text features. Some features provided by CKEditor 5 will thus work as intended only when output to HTML as they have no Markdown equivalent.
```

```

## Known issues

> * Pasting Markdown-formatted content will not automatically convert the pasted syntax markers into properly formatted content currently. See these GitHub issues for possible future development: [#2321](https://github.com/ckeditor/ckeditor5/issues/2321), [#2322](https://github.com/ckeditor/ckeditor5/issues/2322).

Reviewer: ```suggestion
While the Markdown plugin is stable and ready to use, some issues were reported for it. Feel free to upvote 👍 them on GitHub if they are important for you:

* Pasting Markdown-formatted content does not automatically convert the pasted syntax markers into properly formatted content. See these GitHub issues for possible future development: [#2321](https://github.com/ckeditor/ckeditor5/issues/2321), [#2322](https://github.com/ckeditor/ckeditor5/issues/2322).
```

> * The horizontal rule is not supported by the {@link features/autoformat autoformatting} but it does get output if inserted via WYSIWYG interface.

Reviewer: ```suggestion
* The horizontal rule is not supported by the {@link features/autoformat autoformatting} feature but it gets output to Markdown if inserted using the toolbar button.
```

<info-box info>

> 	Please bear in mind that the Markdown output processor will not produce any rich-text content. The [Markdown syntax](https://daringfireball.net/projects/markdown/syntax) is very simple and only supports limited formatting.

Reviewer: ```suggestion
	Please bear in mind that the Markdown data processor does not support all rich text features. The [Markdown syntax](https://daringfireball.net/projects/markdown/syntax) is very simple and only supports limited formatting options.
```

> 	This means than advanced formatting, like list styles, tables or page break markers will be stripped in the effecting data. These are not supported by Markdown and therefore cannot be reproduced. This is not a bug.

Reviewer: ```suggestion
	This means that advanced formatting like list styles, table styles or page break markers will be stripped in the effecting data. These are not supported by Markdown and therefore cannot be converted from HTML to Markdown.
```

```

## Known issues

<info-box info>
	Please bear in mind that the Markdown data processor does not support all rich text features. The [Markdown syntax](https://daringfireball.net/projects/markdown/syntax) is very simple and only supports limited formatting options.

	This means that advanced formatting like list styles, table styles or page break markers will be stripped in the effecting data. These are not supported by Markdown and therefore cannot be converted from HTML to Markdown.
</info-box>

While the Markdown plugin is stable and ready to use, some issues were reported for it. Feel free to upvote 👍&nbsp; them on GitHub if they are important for you:

> * The horizontal rule is not yet supported by the {@link features/autoformat autoformatting} feature but it gets output to Markdown nevertheless if inserted using the toolbar button.

Reviewer: Issue number?

Author: ```suggestion
* The horizontal rule is not yet supported by the {@link features/autoformat autoformatting} feature but it gets output to Markdown nevertheless if inserted using the toolbar button. GitHub issue: [#5720](https://github.com/ckeditor/ckeditor5/issues/5720).
```

Author: Added.

> * Pasting Markdown-formatted content does not automatically convert the pasted syntax markers into properly formatted content. See these GitHub issues for possible future development: [#2321](https://github.com/ckeditor/ckeditor5/issues/2321), [#2322](https://github.com/ckeditor/ckeditor5/issues/2322).

Reviewer: ```suggestion
* Pasting Markdown-formatted content does not automatically convert the pasted syntax markers into properly formatted content. GitHub issues: [#2321](https://github.com/ckeditor/ckeditor5/issues/2321), [#2322](https://github.com/ckeditor/ckeditor5/issues/2322).
```

> * Tables would render as HTML output if there is no table head defined. See the current status of this issue on GitHub: [#8575](https://github.com/ckeditor/ckeditor5/issues/8572).

Reviewer: ```suggestion
* Tables will render as HTML output if there is no table head defined. GitHub issue: [#8575](https://github.com/ckeditor/ckeditor5/issues/8572).
```
